src/arch/xtensa/exc-dump.S

```diff
@@ -29,6 +29,7 @@
 
 #include <arch/offset-defs.h>
 #include <xtensa/corebits.h>
+#include <config.h>
```

Comment:
Not needed.

Reply:
> Not needed.

@tlauda Indeed. Initially, I've added this because I was using CONFIG_IMX8 but then I noticed the XCHAL_INTLEVEL6_MASK. Will fix in v2.